### PR TITLE
Add cookies clearing

### LINKED ISSUE #268: Add ability to clear cookies

### PR DESCRIPTION
**Pull request by EtDu**
**Description**

Adds the ability to clear cookies under Security & Privacy settings.

**Issue**

Resolves #268

### CODE REVIEW

**Review by estebanmino — COMMENTED** [review]
are we storing cookies using `react-native-community/cookies`? it feels like this pr adds something that we're not using. I took a quick look to https://github.com/react-native-community/cookies

**Comment by EtDu** [comment]
When I did a `getAll()` I found a cookie set by OpenSea. I think the library is just a convenient way to access the cookie storage that's being set elsewhere

**Comment by andrepimenta** [comment]
Code looks good to me. But let's merge it after the browser refactoring as discussed

**Review by nicc938 — APPROVED** [review]

File: ios/Podfile.lock [hunk 1]
```diff
@@ -686,4 +692,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 038ce06ee40546e81f34f1057e015a7a12547d35
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.1
```
Comment (rickycodes) [inline ios/Podfile.lock:695]: seems odd that this is getting downgraded? :thinking:

**Review by rickycodes — APPROVED** [review]
LGTM, just curious about COCOAPODS version change

File: ios/Podfile.lock (hunk 1 above)
Comment (EtDu) [reply ios/Podfile.lock:695]: I have no clue, must be a mistake

Comment (EtDu) [reply ios/Podfile.lock]: fixed

**Comment by nicc938** [comment]
ser yes ser thk ser

**Review by nicc938 — APPROVED** [review]
Yes​ser

**Pull request merged**